types/vitest.d.ts

```diff
@@ -1,11 +1,20 @@
-import {type expect} from 'vitest'
+/* eslint-disable @typescript-eslint/no-shadow */
+
+import {
+  type Assertion,
+  type AsymmetricMatchersContaining,
```

Comment:
These two are unused.

You actually type out the name but don't use it in the line 14 and 19.